src/components/Settings/SettingsMain.tsx

```diff
@@ -219,6 +226,28 @@ const SettingsMain: React.FC = () => {
                     )}
                   </div>
                 </div>
+                <div className="form-row">
+                  <label htmlFor="region" className="text-label">
+                    {intl.formatMessage(messages.region)}
+                  </label>
+                  <div className="form-input">
+                    <div className="flex max-w-lg rounded-md shadow-sm">
+                      <Field as="select" id="region" name="region">
```

Comment:
This probably should be a multi-select field, so that users can select more than just one region, or no region at all if they want all results.

Reply:
No region is possible but does TMDB accept multiple regions?

Reply:
Based on the api docs, it doesn't seem like it. I think we would have to make multiple requests to get results for more than one region.